recipes-kernel/linux/linux-intel-acrn_5.4.inc

```diff
@@ -20,3 +20,6 @@ KERNEL_FEATURES_append = "features/netfilter/netfilter.scc \
                           cfg/hv-guest.scc \
                           cfg/paravirt_kvm.scc \
 "
+# Kernel config 'CONFIG_GPIO_LYNXPOINT' goes by a different name 'CONFIG_PINCTRL_LYNXPOINT' in
+# linux-intel. This cause warning during kernel config audit. So suppress the harmless warning for now.
+KCONF_BSP_AUDIT_LEVEL = "0"
```

Comment:
I believe this resolves @saininav concern (which I am glad you brought up as I saw this on my own experimental branch).